### PR TITLE
Feature: Add support for base-goerli

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Please include a summary of the change and be sure you follow the contributions rules we do provide [here](./CONTRIBUTIONS.md)

Task ID: [OS-620](https://aragonassociation.atlassian.net/browse/OS-620)
Task ID: [OS-621](https://aragonassociation.atlassian.net/browse/OS-621)
Task ID: [OS-628](https://aragonassociation.atlassian.net/browse/OS-628)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them when possible.
- [ ] I have updated the `CHANGELOG.md` file in the root folder of the package after the `[UPCOMING]` title and before the latest version.
- [ ] I have tested my code on the test network.

[OS-620]: https://aragonassociation.atlassian.net/browse/OS-620?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[OS-621]: https://aragonassociation.atlassian.net/browse/OS-621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[OS-628]: https://aragonassociation.atlassian.net/browse/OS-628?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10,13 +10,20 @@
     "@jridgewell/gen-mapping" "^0.1.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@aragon/osx-ethers@1.3.0-rc0", "@aragon/osx-ethers@^1.3.0-rc0":
+"@aragon/osx-ethers@1.3.0-rc0":
```
Comment (heueristik) [inline yarn.lock]: Why do we have two versions here?

File: modules/client-common/test/context.test.ts [hunk 2]
```diff
@@ -246,4 +246,31 @@ describe("Context instances", () => {
       LIVE_CONTRACTS.matic.ensRegistry,
     );
   });
+  it("Should create a context with baseGoerli as network and have the correct values", () => {
```
Comment (heueristik) [inline modules/client-common/test/context.test.ts:249]: Is there a way how we can do a bit more probing to make sure that some standard methods work with Base?
Exhaustive tests are certainly nothing for this PR, but having some probing tests could be helpful before you do E2E testing for the entire codebase.

**Review by heueristik — COMMENTED** [review]
Looks good! Two questions regarding tests and versioning.

File: yarn.lock (hunk 1 above)
Comment (josemarinas) [reply yarn.lock]: Idk, there was 2 now there is only one which I believe is the correct one

File: modules/client-common/test/context.test.ts (hunk 2 above)
Comment (josemarinas) [reply modules/client-common/test/context.test.ts:249]: I created another test for testing that a client with the base context can be ccreated. I believe there isn't much we can do rn without doing a proper e2e test.

File: yarn.lock (hunk 1 above)
Comment (heueristik) [reply yarn.lock]: > there is only one which I believe is the correct one

We now have 
`"@aragon/osx-ethers@1.3.0-rc0":`
and
`"@aragon/osx-ethers@^1.3.0-rc0.1":`
as dependencies.

Not saying this is important, but just makes me wonder why. Go ahead

Comment (josemarinas) [reply yarn.lock]: no, you are absolutely right, I forgot the update the osx-ethers version on the client package.json, thank you

File: modules/client-common/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client-common",
   "author": "Aragon Association",
-  "version": "1.2.1-rc0",
+  "version": "1.3.0-rc0",
```
Comment (heueristik) [inline modules/client-common/package.json:4]: Double check, if this is the right version number

**Review by mathewmeconry — DISMISSED** [review]
LGTM

**Review by heueristik — DISMISSED** [review]

**Review by heueristik — APPROVED** [review]

**Pull request merged**